eras/alonzo/test-suite/cardano-ledger-alonzo-test.cabal

```
@@ -46,7 +46,7 @@ library
         bytestring,
         cardano-ledger-alonzo:{cardano-ledger-alonzo, testlib} >=1.7,
         cardano-ledger-binary:{cardano-ledger-binary, testlib} >=1.0,
-        cardano-ledger-core:{cardano-ledger-core, testlib} >=1.11 && <1.15,
+        cardano-ledger-core:{cardano-ledger-core, testlib} >=1.11 && <1.16,
```

This package does not need to change any dependencies
```suggestion
        cardano-ledger-core:{cardano-ledger-core, testlib} >=1.11 && <1.15,
```